queries/dart/highlights.scm

```diff
@@ -19,7 +19,7 @@
   "$" @punctuation.special
   "{" @punctuation.special
   "}" @punctuation.special
-) @embedded
```

Comment:
`@embedded` is not really correct here since we don't want to inject another Dart parser here. The orignal one has already all the children in the same AST. Or we could change `@embedded` to reset all highlights.